docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md

```diff
@@ -91,6 +91,13 @@ There are two options to authenticate the Connector with AWS:
 
 <TabItem value='inbound'>
 
+:::note
+To ensure consistent and stable work of Amazon Simple Queue Service Connector, it is not recommended
```

Comment:
```suggestion
To ensure consistent and stable behaviour of the Amazon Simple Queue Service Connector, it is not recommended
```
Maybe @christinaausley has a better idea for phrasing this.

Reply:
to maintain stable behavior from the Amazon SQS Connector, do not subscribe multiple Amazon SQS Connectors to the same queue.

WDYT?

Reply:
@christinaausley perfect!

Reply:
Adjusted in latest commit 👍